pkg/clusterd/disk.go

```diff
@@ -176,5 +197,15 @@ func PopulateDeviceUdevInfo(d string, executor exec.Executor, disk *sys.LocalDis
 		disk.WWN = val
 	}
 
+	if disk.Type == sys.PartType {
+		if val, ok := udevInfo["PARTNAME"]; ok {
+			disk.Label = val
+		}
+
+		if val, ok := udevInfo["ID_PART_ENTRY_NAME"]; ok {
```

Comment:
is it intended for both the PARTNAME and ID_PART_ENTRY_NAME to set the `disk.Label`? Do we only expect one of the properties to be set, so there shouldn't be a conflict? Or they are the same value?

Reply:
I guess the goal is to choose one or the other and potentially overwrite. So both provide the same value anyway.